toolkit/templates/character_generator.html

```diff
@@ -3,4 +3,651 @@
 {% load static %}
 
 {% block application_pane %}
+  <div class="character-header">
+    <img src
+         {% static "Character_Header.png" %}
+         alt="Character Header"
+         style="width: 45%"/>
+  </div>
+  <div class="character-row">
+    <div class="character-name">
+      <label>
+        <h4>Character Name</h4>
+        <input class="big"
```

Comment:
may be worth using bootstrap's input boxes to conform with the rest of the website https://getbootstrap.com/docs/5.2/forms/input-group/#sizing